tests/functional/v1/test_leasing_positive.py

```diff
@@ -0,0 +1,49 @@
+import tests.functional.base as base
+
+class PositiveLeasingTests(base.EsiFunctionalBase):
+    @classmethod
+    def setUpClass(cls):
+        super(PositiveLeasingTests, cls).setUpClass()
+        cls._init_dummy_cloud(cls, 'owner', 'owner')
+        cls._init_dummy_cloud(cls, 'lessee', 'lessee')
+
+        cls._init_client(cls, 'owner')
+        cls._init_client(cls, 'lessee')
```

Comment:
if _init_dummy_cloud and _init_client will always be paired, I'd suggest creating a new _init function that calls the two.

Reply:
I think I'll have _init_dummy_cloud call _init_client internally then. (I also use _init_client() to initialize the admin client that sets up the dummy clouds)